catalog/video/video-mpegts-stream/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -33,9 +33,9 @@
                 type="String" default="" />
 
         <AD
-                description="Maximum file size (bytes) before rollover. Must be >=1."
+                description="Maximum file size (megabytes) before rollover. Must be >=1."
```

Comment:
Agreed.  It's worth noting that the video admin plugin drops the decimal values and uses the whole number